cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -397,58 +384,45 @@ components:
             enum:
               - "cell"
               - "nucleus"
-          - # If assay_ontology_term_id is EFO:0009918 or its descendants, 'suspension_type' MUST be 'na'
+          - # If assay_ontology_term_id is EFO:0008994 or its descendants, 'suspension_type' MUST be 'na'
             complex_rule:
               match_ancestors:
                 column: assay_ontology_term_id
                 ancestors:
                   EFO:
-                    - EFO:0009918
+                    - EFO:0008994
                 inclusive: True
             type: categorical
             error_message_suffix: >-
-              when 'assay_ontology_term_id' is EFO:0009918 or its descendants
+              when 'assay_ontology_term_id' is EFO:0008994 or its descendants
             enum:
               - "na"
-          - # If assay_ontology_term_id is EFO:0700000 or its descendants, 'suspension_type' MUST be 'na'
+          - # If assay_ontology_term_id is EFO:0008919 or its descendants, 'suspension_type' MUST be 'cell'
             complex_rule:
               match_ancestors:
                 column: assay_ontology_term_id
                 ancestors:
                   EFO:
-                    - EFO:0700000
+                    - EFO:0008919
                 inclusive: True
             type: categorical
             error_message_suffix: >-
-              when 'assay_ontology_term_id' is EFO:0700000 or its descendants
+              when 'assay_ontology_term_id' is EFO:0008919 or its descendants
             enum:
-              - "na"
-          - # If assay_ontology_term_id is EFO:0008994 or its descendants, 'suspension_type' MUST be 'na'
+              - "cell"
+          - # If assay_ontology_term_id is EFO:0002761 or its descendants, 'suspension_type' MUST be 'nucleus'
             complex_rule:
               match_ancestors:
                 column: assay_ontology_term_id
                 ancestors:
                   EFO:
-                    - EFO:0008994
+                    - EFO:0002761
                 inclusive: True
             type: categorical
             error_message_suffix: >-
-              when 'assay_ontology_term_id' is EFO:0008994 or its descendants
+              when 'assay_ontology_term_id' is EFO:0002761 or its descendants
             enum:
-              - "na"
-          - # If assay_ontology_term_id is EFO:0008919 or its descendants, 'suspension_type' MUST be 'cell'
```

Comment:
is this an extra change for this PR? I don't see it in the list of ids to delete.

Reply:
it's still [here](https://github.com/chanzuckerberg/single-cell-curation/pull/1007/files#diff-7cd17d19c6ca9ca666aeb04b33d9cc5b5e46100a05bd3076ed3730dfc720af3fR400)!

Not sure why the diff is showing it as deleted, I didn't touch it. Just appended below it

Reply:
I had a feeling that's what it was. Thank you for confirming.